packages/aws/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.42.1"
```

```suggestion
- version: "2.43.0"
```
Enhancements bump minor.